api-2.2/src/main/java/org/openmrs/module/fhir/helper/ConditionHelperImpl2_2.java

```diff
@@ -65,6 +68,62 @@ public Condition createCondition(Condition condition) {
 		return generateFHIRCondition(openMrsCondition);
 	}
 
+	/**
+	 * @see org.openmrs.module.fhir.api.helper.ConditionHelper#searchConditionByPatientUuid(java.lang.String)
+	 */
+	@Override
+	public List<Condition> searchConditionByPatientUuid(String patientUuid) {
+		List<Condition> fhirConditions = new ArrayList<>();
+		PatientService patientService = Context.getPatientService();
+		ConditionService conditionService = Context.getService(org.openmrs.api.ConditionService.class);
+
+		if (StringUtils.isNotBlank(patientUuid)) {
+			conditionService.getActiveConditions(
+					patientService.getPatientByUuid(patientUuid))
+					.forEach(condition -> fhirConditions.add(generateFHIRCondition(condition)));
+		}
+
+		return fhirConditions;
+	}
+
+	/**
+	 * @see org.openmrs.module.fhir.api.helper.ConditionHelper#searchConditionByPatientIdentifier(java.lang.String)
+	 */
+	@Override
+	public List<Condition> searchConditionByPatientIdentifier(String identifier) {
+		List<Condition> conditions = new ArrayList<>();
+		PatientService patientService = Context.getPatientService();
+		List<org.openmrs.Patient> patientList = patientService.getPatients(null,
+				identifier, null, true);
+
+		ConditionService conditionService = Context.getService(ConditionService.class);
+
+		if (patientList != null && !patientList.isEmpty()) {
+			patientList.stream().forEach(patient -> {
+				conditions.addAll(conditionService.getActiveConditions(patient)
+						.stream()
+						.map(this::generateFHIRCondition)
+						.collect(Collectors.toList()));
+			});
+		}
+
+		return conditions;
+	}
+
+	/**
+	 * @see org.openmrs.module.fhir.api.helper.ConditionHelper#searchConditionByUuid(java.lang.String)
+	 */
+	@Override
+	public Condition searchConditionByUuid(String uuid) {
```

Comment:
getConditionByUuid

Reply:
We have get condition and search condition by uuid. They are different.
getConditionByUuid `/ws/fhir/Condition/<UUID>`
searchConditionByUuid `/ws/fhir/Condition?_id= <UUID>` .

Reply:
Why wouldn't the two translate to getConditionByUuid?

Reply:
done